asv/plugins/git.py

```diff
@@ -19,20 +19,26 @@
 class Git(Repo):
     dvcs = "git"
 
-    def __init__(self, url, path, _checkout_copy=False):
+    def __init__(self, url, mirror_path):
         self._git = util.which("git")
-        self._path = os.path.abspath(path)
+        self._path = os.path.abspath(mirror_path)
         self._pulled = False
 
-        if not os.path.isdir(self._path):
-            args = ['clone']
-            if _checkout_copy:
-                args.append('--shared')
-            else:
-                log.info("Cloning project")
-                args.append('--mirror')
-            args.extend([url, self._path])
-            self._run_git(args, chdir=False)
+        if Git.is_local_repo(url):
```

Comment:
Any reason to do this rather than `self.is_local_repo`...?  This would seem to make subclassing harder.